pkg/cloudprovider/types.go

```diff
@@ -82,8 +82,19 @@ type CapacityConstraints struct {
 	Architecture Architecture
 }
 
-// CapacityPacking is a solution to packing pods onto nodes given constraints.
-type CapacityPacking map[*v1.Node][]*v1.Pod
+// PodPackings is a solution to packing pods onto nodes given constraints.
+type PodPackings map[*v1.Node][]*v1.Pod
+
+type Packings struct {
```

Comment:
IIUC, this struct is used internally by the AWS cloud provider implementation and probably belongs in aws/fleet/packer.go. 

This cloudprovider types file should be as simple as possible and represent the minimal structs and interfaces necessary to implement a cloud provider.